packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -546,6 +546,8 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
         const { combinedRotation, textBaseline, textAlign, range } = params;
         const text = datum.tickLabel;
         const sideFlag = label.getSideFlag();
+        // TODO: Set tickSize to zero when tick.enabled = false
+        // https://ag-grid.atlassian.net/browse/AG-10059
```

Comment:
FYI: I'm happy with a condensed format similar to `// @todo(AG-10059): Set tickSize to zero when tick.enabled = false` for this sort of comment.